script/tool/lib/src/build_examples_command.dart

```diff
@@ -121,6 +134,14 @@ class BuildExamplesCommand extends PackageLoopingCommand {
           'were specified. At least one platform must be provided.');
       throw ToolExit(_exitNoPlatformFlags);
     }
+
+    if (usingSwiftPackageManager) {
+      await processRunner.runAndStream(
+        flutterCommand,
+        <String>['config', '--enable-swift-package-manager'],
+        exitOnError: true,
```

Comment:
This will explode on our `stable` tests (which you aren't seeing in presubmit because we only run native `stable` tests on post-submit to reduce CI utilization). You'll need to gate either the whole call, or just exiting on failure, on a `Platform.environment['CHANNEL'] != 'stable'` check for now, with a TODO to enable it for stable once the flag is available there.

Reply:
Oh good catch, done and validated: https://chromium-swarm.appspot.com/task?id=6929f2858af7f110